src/styles/index.scss

```diff
@@ -1,3 +1,221 @@
+$primary-color: #00acdc;
+$secondary-color: #616070;
+$tertiary-color: #060b35;
+$font-family: 'Roboto', sans-serif;
+
 body {
   margin: 0;
+  font-family: $font-family;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
```

Comment:
I'd prefer to use gap property whenever it's possible instead of margins.
As a general rule of thumb, you always want a parent component being responsible for child elements positioning.